frontend/public/components/utils/_file-input.scss

```diff
@@ -15,5 +15,35 @@
     right: 0;
     text-align: right;
     top: 0;
+    z-index: 1000;
+  }
+}
+
+.co-file-dropzone {
+  position: relative;
+  .dropzone {
+    border: 3px solid #fff;
+    background: rgba(0,0,0,0.3);
+    bottom: 0;
+    color: #fff;
+    left: 0;
+    margin: -3px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: 1001;
+    &.drop-over {
+      border-color: $color-pf-blue-300;
+      color: $color-pf-blue-300;
+    }
+    .drop-text {
```

Comment:
Better to use BEM naming for these classes. (I think we're setting the class names ourselves?)